src/repository/BaseMongoDb.js

```diff
@@ -88,14 +88,18 @@ class BaseMongoDb implements IBaseDatabase {
       await collection.remove(this.__translateQuery({ _id: id })),
   );
 
+  // eslint-disable-next-line no-unused-vars
+  __filterID = ({ id, ...otherProps }: Object): Object => ({ ...otherProps });
```

Comment:
add this basically for update User, so it won't save redundant `id` field to the database.